content/riak/ts/1.3.0/using/querying.md

```diff
@@ -53,7 +53,9 @@ The timestamp in the partition key is an integer (in milliseconds) that must be
 
 ### Local Key
 
-Column names that are in the local key but not in the partition key are not required for the query.
+For any field in the local key that is not in the partition key, your
+query can include any operator supported for that field's type and
+bounded ranges are not required.
```

Comment:
I'm trying not to be a stickler about it here, but we keep jumping back and forth between respecting an 80 column width, and not. The majority of this page is not respecting the 80 column width, but this line is, and there's a really jerky corner of my head shouting "Consistency is King!" over and over again.